simplq/src/styles/homePage.module.scss

```diff
@@ -143,3 +143,106 @@
     font-size: small;
   }
 }
+
```

Comment:
We try to use `rem` instead of `px` where ever possible. Can you convert?

Reply:
Okay, I will change the things you asked for.

Thank you.